src/chrome/content/rules/Fefe.xml

```diff
@@ -31,17 +30,23 @@
 	^fefe.de doesn't exist.
 
 -->
+<ruleset name="Fefe.de (partial)" platform="Lets Encrypt">
+  <target host="blog.fefe.de" />
+        <target host="ptrace.fefe.de" />
+        <target host="www.fefe.de" />
+
+        <rule from="^http:"
+                to="https:" />
+</ruleset>
```

Comment:
as previous [comment](https://github.com/EFForg/https-everywhere/pull/3521#issuecomment-165909189) pointed out, remove line 33, 38-40. Then put line 34-36 below line 42. `platform` argument is not necessary in this case.